doc/api/vm.md

```diff
@@ -148,53 +184,59 @@ console.log(globalVar);
 // 1000
 ```
 
-The options for running a script are:
-
-- `filename`: allows you to control the filename that shows up in any stack
-  traces produced.
-- `lineOffset`: allows you to add an offset to the line number that is
-  displayed in stack traces
-- `columnOffset`: allows you to add an offset to the column number that is
-  displayed in stack traces
-- `displayErrors`: if `true`, on error, attach the line of code that caused
-  the error to the stack trace. Applies only to runtime errors executing the
-  code; it is impossible to create a `Script` instance with syntax errors, as
-  the constructor will throw.
-- `timeout`: a number of milliseconds to execute the script before terminating
-  execution. If execution is terminated, an [`Error`][] will be thrown.
-
 ## vm.createContext([sandbox])
 
-If given a `sandbox` object, will "contextify" that sandbox so that it can be
-used in calls to [`vm.runInContext()`][] or [`script.runInContext()`][]. Inside
-scripts run as such, `sandbox` will be the global object, retaining all its
-existing properties but also having the built-in objects and functions any
-standard [global object][] has. Outside of scripts run by the vm module,
-`sandbox` will be unchanged.
+* `sandbox` {Object}
 
-If not given a sandbox object, returns a new, empty contextified sandbox object
-you can use.
+If given a `sandbox` object, The `vm.createContext()` method will "contextify"
```

Comment:
yeah, this uses "contextified" quite a bit without any real context of what that means. I was stewing over what to do about that.